python/sqlalchemy/security/sqlalchemy-execute-raw-query.py

```diff
@@ -6,7 +6,7 @@
 engine = create_engine('postgresql://user@localhost/database')
 echo("database connexion: ok")
 # ruleid: sqlalchemy-execute-raw-query
-engine.execute("INSERT INTO person (name) VALUES ('" + name + "')")
+engine.execute("INSERT INTO person (name) VALUES ('" + name + "')", multi=False)
```

Comment:
Can you make this a completely new `# ruleid: ...` test? That way we have both tests confirming behavior :)